test/controllers/declaration/DeclarationHolderSummaryControllerSpec.scala

```diff
@@ -122,8 +123,19 @@ class DeclarationHolderSummaryControllerSpec extends ControllerSpec with OptionV
       }
     }
 
-    onJourney(CLEARANCE, OCCASIONAL, STANDARD, SUPPLEMENTARY, SIMPLIFIED) { implicit request =>
-      "return 303 (SEE_OTHER)" when {
+    "return 303 (SEE_OTHER)" when {
```

Comment:
Am I correct that `STANDARD` declaration scenario is not tested?

Reply:
Ah yes, must of forgotten this one.